flink-doris-connector/src/main/java/org/apache/doris/flink/rest/RestService.java

```diff
@@ -234,6 +234,38 @@ public static String parseResponse(HttpURLConnection connection, Logger logger)
         }
     }
 
+    @VisibleForTesting
+    public static String parseFlightSql(
+            DorisReadOptions readOptions,
+            DorisOptions options,
+            PartitionDefinition partition,
+            Logger logger)
+            throws IllegalArgumentException {
+        String[] tableIdentifiers = parseIdentifier(options.getTableIdentifier(), logger);
+        String readFields =
+                StringUtils.isBlank(readOptions.getReadFields())
+                        ? "*"
+                        : readOptions.getReadFields();
+        String sql =
+                "select "
+                        + readFields
+                        + " from `"
+                        + tableIdentifiers[0]
+                        + "`.`"
+                        + tableIdentifiers[1]
+                        + "`";
+        String tablet =
+                partition.getTabletIds().stream()
+                        .map(Object::toString)
+                        .collect(Collectors.joining(","));
+        sql += "  TABLET(" + tablet + ") ";
```

Comment:
Why does the tablet need to be spliced ​​in here?

Reply:
In DorisSource.createEnumerator() method, readers split it by node. Currently, doris does not support specifying be node to read data for the time being, so you need to specify a tablet to query